clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/configuration/ConfigClientSslProperties.groovy

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Pivotal, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.configuration
+
+import org.springframework.boot.context.embedded.Ssl
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.NestedConfigurationProperty
+
+/**
+ * Use the following properties to enable SSL and configure a certificate store.
+ *
+ * See bootstrap.yml for commented example.
+ */
+@ConfigurationProperties(prefix = "config-client")
+class ConfigClientSslProperties {
```

Comment:
I'd actually prefer (though it isn't exactly accurately named) to see the use of https://github.com/spinnaker/kork/blob/master/kork-web/src/main/groovy/com/netflix/spinnaker/okhttp/OkHttpClientConfigurationProperties.groovy bean for the SSL configuration. If someone is using SSL they will have configured that bean, and it would suck to have to duplicate the same set of configuration into this bean